ql/if/queryplan.thrift

```diff
@@ -111,7 +111,8 @@ enum StageType {
   RANGER_DUMP,
   RANGER_LOAD,
   ATLAS_DUMP,
-  ATLAS_LOAD
+  ATLAS_LOAD,
+  COPY_LOG
```

Comment:
StatgeType for ReplStateLogTask is already defined: REPL_STATE_LOG
Do we still need this?

Reply:
Done